<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Files.qrc">
    <normaloff>:/PadImg</normaloff>:/PadImg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(50, 50, 50, 255));
	color:#FFFFFF;
}
QLabel, QPushButton, QToolButton, QTextEdit, QTabWidget {
    font: 666 14pt &quot;Cantarell&quot;;
}
QPushButton, #resetRememberLocation, #resetEnableTaskbar, 
  #resetShowSong, #resetStartInTaskbar, #resetStream, #resetMetaData {
	background-color:#47A;
}
QToolButton {
	background-color: #78cceb;
}
QTabBar::tab {
	background-color: #69C;
}
QTabBar::tab {
	border: 2px solid #C4C4C3;
	border-bottom-color: #C2C7CB; 
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	min-width: 8ex;
	padding: 2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
	background-color:#47A;
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; 
}

QTabBar::tab:!selected {
    margin-top: 3px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="noteSettings">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Settings (Restart to apply)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="window">
      <attribute name="title">
       <string>Window</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="RememberLocationLayout">
         <item>
          <widget class="QToolButton" name="checkRememberLocation">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <selectedoff>:/CheckUnchecked</selectedoff>
             <selectedon>:/CheckChecked</selectedon>
            </iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteRememberLocation">
           <property name="text">
            <string>Remember window location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="resetRememberLocation">
           <property name="text">
            <string>&lt;-</string>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resetWindowLayout"/>
       </item>
       <item>
        <spacer name="verticalSpacerWindow">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="taskBar">
      <attribute name="title">
       <string>TaskBar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="enableTaskbarLayout">
         <item>
          <widget class="QToolButton" name="checkEnableTaskbar">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <selectedoff>:/CheckUnchecked</selectedoff>
             <selectedon>:/CheckChecked</selectedon>
            </iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteEnableTaskbar">
           <property name="text">
            <string>Enable Taskbar icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="resetEnableTaskbar">
           <property name="text">
            <string/>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="showSongLayout">
         <item>
          <widget class="QToolButton" name="checkShowSong">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <selectedoff>:/CheckUnchecked</selectedoff>
             <selectedon>:/CheckChecked</selectedon>
            </iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteShowSong">
           <property name="text">
            <string>Show song info in taskbar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="resetShowSong">
           <property name="text">
            <string/>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="startInTaskbarLayout">
         <item>
          <widget class="QToolButton" name="checkStartInTaskbar">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <selectedoff>:/CheckUnchecked</selectedoff>
             <selectedon>:/CheckChecked</selectedon>
            </iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteStartInTaskbar">
           <property name="text">
            <string>Start PadRadio in taskbar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="resetStartInTaskbar">
           <property name="text">
            <string/>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacerTaskbar">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="servers">
      <attribute name="title">
       <string>Servers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="streamLayout">
         <item>
          <widget class="QLabel" name="noteStream">
           <property name="text">
            <string>Stream URL: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editStream"/>
         </item>
         <item>
          <widget class="QToolButton" name="resetStream">
           <property name="text">
            <string/>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MetaDataLayout">
         <item>
          <widget class="QLabel" name="noteMetaData">
           <property name="text">
            <string>MetaData URL: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editMetaData"/>
         </item>
         <item>
          <widget class="QToolButton" name="resetMetaData">
           <property name="text">
            <string/>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacerServer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close Window</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Files.qrc"/>
 </resources>
 <connections/>
</ui>
